homeassistant/components/media_player/plex.py

```diff
@@ -62,6 +107,37 @@ def config_from_file(filename, config=None):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup the Plex platform."""
+    # optional parameters
+    optional_config = {}
+    optional_config["entity_namespace"] = config.get("entity_namespace")
+    optional_config[CONF_INCLUDE_NON_CLIENTS] = config.get(
+        CONF_INCLUDE_NON_CLIENTS)
+    optional_config[CONF_USE_EPISODE_ART] = config.get(CONF_USE_EPISODE_ART)
+    optional_config[CONF_USE_CUSTOM_ENTITY_IDS] = config.get(
+        CONF_USE_CUSTOM_ENTITY_IDS)
+    optional_config[CONF_SHOW_ALL_CONTROLS] = config.get(
+        CONF_SHOW_ALL_CONTROLS)
+
+    max_frozen_playing = DEFAULT_MAX_FROZEN_PLAYING
+    try:
+        max_frozen_playing = int(config.get(CONF_MAX_FROZEN_PLAYING))
+    except ValueError:
+        pass
+    finally:
```

Comment:
This is also wrong. If `config.get(CONF_MAX_FROZEN_PLAYING)` was a string, you will now run into bad code because finally is always executed although coercion failed.